roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -17,6 +17,27 @@
 - name: Get ceph_cli
   ansible.builtin.include_tasks: ceph_cli.yml
 
+- name: Check if swift endpoint is already created
+  shell: "oc rsh openstackclient openstack endpoint list | grep 'swift.*object-store' | wc -l"
+  register: swift_endpoint
+  delegate_to: localhost
+
+- name: Configure ceph rgw as backend to Swift
```

Comment:
I may be wrong, but this doesn't configure Ceph RGW as backend for Swift, but it configures RGW to expose the same Object Storage API that Swift exposes. It is a compatible implementation of the same API, not a backend..
If tha's the case, please fix the task name and the commit message and the PR message.

Reply:
I think the intention here is to configure an external Ceph Object Gateway (RGW) as an Object Store service and Swift client tool can be used to interact with object store.

I just copied the commands and task name from [1] , @fmount can you confirm on this ?

[1] https://github.com/openstack-k8s-operators/docs/blob/main/ceph.md#configure-swift-with-a-rgw-backend

Reply:
I think we can improve the wording to be something like: `Configuring Ceph Object Store to use external Ceph Object Gateway`, including the doc patch. We commonly refer to rgw as a backend, but technically we're not setting up any backend for the swift service, we're just replacing it w/ RGW.
@katarimanojk I think it's good we can improve the wording. I can do that in the docs you mentioned, but +1 to move to a better wording.